library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -73,15 +73,10 @@ return_status libspdm_set_data(IN void *context, IN libspdm_data_type_t data_typ
 
     switch (data_type) {
     case LIBSPDM_DATA_SPDM_VERSION:
-        if (data_size >
-            sizeof(spdm_version_number_t) * SPDM_MAX_VERSION_COUNT) {
-            return RETURN_INVALID_PARAMETER;
-        }
+        ASSERT (data_size <= sizeof(spdm_version_number_t) * SPDM_MAX_VERSION_COUNT);
         if (parameter->location == LIBSPDM_DATA_LOCATION_CONNECTION) {
-            if (data_size != sizeof(spdm_version_number_t)) {
-                /* Only have one connected version */
-                return RETURN_INVALID_PARAMETER;
-            }
+            /* Only have one connected version */
```

Comment:
How does integrator specify that they support both 1.0 and 1.1?

Reply:
this is version for connection. We can only choose 1.0 or 1.1.
Not local provision.

Reply:
Ah yeah that makes sense.